install/kubernetes/cilium/README.md

```diff
@@ -129,6 +129,8 @@ contributors across the globe, there is almost always someone available to help.
 | encryption.secretName | string | `"cilium-ipsec-keys"` | Name of the Kubernetes secret containing the encryption keys. This option is only effective when encryption.type is set to ipsec. |
 | encryption.type | string | `"ipsec"` | Encryption method. Can be either ipsec or wireguard. |
 | endpointHealthChecking.enabled | bool | `true` |  |
+| endpointStatus.enabled | bool | `false` | Enable endpoint status |
+| endpointStatus.status | string | `policy` | Endpoint status. Can be policy, health, controllers, logs or state |
```

Comment:
If it can be all of those parameters, why does it default to policy?

Reply:
I set it default to policy cause I was thinking most users will set this option true to enable the columns in the cep command but I'm happy to change to empty and required field so users can select the status they want.

Reply:
Updated PR and description

Reply:
I think we probably _should_ enable policy by default specifically for this reason. It impacts scale, but we have a scalability guide to instruct users how to get the most scalable setup. On the other hand, disabling this by default means that most users just see the CEP output as broken and it's not obvious why.